docs: add NetBox Enterprise release notes (#40)
netboxlabs/console-docs: docs/netbox-enterprise/nbe-release-notes-1.x.md

> # NetBox Enterprise 1.x Release Notes

Reviewer: ```suggestion
# NetBox Enterprise 1.0 Release Notes
```

## 1.0.5

> 1.0.5 adds support for KOTS installs to scrape Prometheus metrics from NetBox, as well as the embedded PostgreSQL, Redis, and SeaweedFS if they are enabled.

Reviewer: ```suggestion
Adds support for KOTS installs to scrape Prometheus metrics from NetBox, as well as the embedded PostgreSQL, Redis, and SeaweedFS if they are enabled.
```

> It also updates NetBox to 4.0.9, and includes some bug fixes and improved startup time.

Reviewer: ```suggestion
Updated to support NetBox v4.0.9, and includes additional bug fixes and startup time improvements.
```

## 1.0.4

> 1.0.4 adds OWASP password complexity validation to NetBox, as well as some dependency updates.

Reviewer: ```suggestion
Adds OWASP password complexity validation to NetBox, and includes dependency updates.
```

## 1.0.3

> 1.0.3 is a small release that fixes some issues with preflight checks, plus some minor dependency updates.

Reviewer: ```suggestion
Fixes issues with preflight checks, and includes minor dependency updates.
```

## 1.0.2

> 1.0.2 is a small update that includes a number of dependency updates and bug fixes, as well as initial alpha support for backups of built-in Redis and PostgreSQL.

Reviewer: ```suggestion
Provides a number of dependency updates and bug fixes, and includes initial support for backups of built-in Redis and PostgreSQL.
```

## 1.0.1

> 1.0.1 is focused on bugs encountered during the initial beta rollout of the NetBox Enterprise application.

Reviewer: ```suggestion
Provides bug fixes encountered during the initial rollout of the NetBox Enterprise application.
```

## 1.0.0

> 1.0.0 is a small update in the Beta stream to do some final cleanup as we prepare for a wider release.

Reviewer: ```suggestion
Provides final cleanup of the Beta stream in preparation for the wider release.
```

> It contains mostly internal changes, plus fixes an issue where annotations could render improperly and cause problems with upgrades.

Reviewer: ```suggestion
Provides a number of internal changes, and includes fixes for an issue where annotations could render improperly and cause problems with upgrades.
```

> It also updates NetBox 4.x to version 4.0.7.

Reviewer: ```suggestion
Updated to support NetBox v4.0.7.
```
